.circleci/config.yml

```diff
@@ -36,10 +36,10 @@ save_cache: &save_cache
   - save_cache:
       key: sbt-actors-cache
       paths:
-          - "~/.ivy2/cache"
```

Comment:
was there a formatting issue with the circle config yaml? Was these formatting changes intentional?

Reply:
I pasted `mdoc`s parts from core `zio` and plain paste required the format but IDE formatted whole file. I thought it doesn't matter but just looked at core `zio` and there's also these lines intended with four instead of two spaces. I barely used circleci and don't know whether it matters. 
Should I revert these changes?

Reply:
Yeah I think lets remove the indentation changes your IDE made from this PR. 

If easier just remove the circle changes out of this PR and we can do that as a separate PR, but up to you.

Reply:
Sure, just reverted all white-space formatting in `config.yml`